.github/workflows/staging-deploy.yml

```
@@ -0,0 +1,112 @@
+        run: |
+          curl -fsSL -X POST "${STAGING_API_BASE_URL}/functions/v1/creatives-text" \
+            -H "Content-Type: application/json" \
+            -d '{"product":"Sneaker X","audience":"runners in Mumbai"}' || exit 1
+
+      # Run Playwright tests
+      - name: Run Playwright tests
+        run: npx playwright test
+
+      # Upload test results
+      - name: Upload test results
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)